### DISCUSSION TITLE
Change global footer

### DISCUSSION THREAD

**Discussion by Nodrogchavez** [pH7Software/pH7-Social-Dating-CMS#839] [Q&A]
I want to change the footer by removing the icons for Facebook etal, and replace that area with the Google Translate widget.  Where is the php file where I can do that?  I have prowled extensively through the files but am unable to find anything relating to that.  Also, is the footer a html area? Any guidance would be appreciated.

**Comment by excalibur-web** [comment]
what version of ph7 ?

**Reply by Nodrogchavez** [reply]
16.0.0

**Comment by polynamaude** [comment]
Look inside the template folder
https://github.com/pH7Software/pH7-Social-Dating-CMS/blob/master/templates/themes/base/tpl/bottom_menu.inc.tpl

Btw, @excalibur-web If you ever fix some code, the least you can do is push back the modification onto a forked repository. That's what's called giving back to community.

**Reply by Nodrogchavez** [reply]
Thanks for your input polynamaude

Nothing in that file about those icons, but in layout.tpl in that same folder there is something that needs me to spend some time figuring out.  It is the flow of the links referencing other links that needs my attention, and I think it is that tpl that I will need to modify.  I am not a programer, but to a fair degree understand the coding (I'm more of a software designer).

**Reply by Nodrogchavez** [reply]
Hmmm... I left a reply, but it seems to have disappeared.  The tpl you referenced does not have anything in it that relates to those icons at the left side of the bottom.  I did however find some links in the layout.tpl that I need to look into.

<div role="contentinfo">
        <div class="ft_copy">
          {{ $design->littleSocialMediaWidgets() }}

          <p>
            &copy; <ph:date value="Y" /> <strong>{site_name}</strong>  {{ $design->link() }}
          </p>
        </div>
        {{ $design->langList() }}
        {main_include 'bottom_menu.inc.tpl'}
      </div>

I'm not a programer, but have designed some software over the years, and hopefully that will be enough for me to get rid of all of those bottom left icons and replace them with the Google Translator widget that I use on other of my websites.

Thanks for the guidance.

**Reply by polynamaude** [reply]
Before investing too much time. Look at the issues open and the bug that never got fixed. Example : The when you reach 2k user, the system will start failing randomnly because of bad design.

I "wasted" much time with this.

**Reply by Nodrogchavez** [reply]
Interesting comment, and even more so as I was rapidly coming to the same conclusion. I had even mentioned that my techie at OSSN in a message to him on Tuesday.

"I must tell you that your OSSN software is much more elegant than that of the OSDS. Configuring the OSDS is very time consuming, and it still has numerous bugs. Also, the guy who wrote it persists in asking for donations/financial help, while at the same time making sure that everyone knows he is looking for a job with a software company. I'll continue playing with it, but am getting the sense that it may never be ready for prime time."

OSSN also has a free version of their social network software.  https://www.softlab24.com

Thanks again for your input.

**Reply by polynamaude** [reply]
@Nodrogchavez He also used to "backlink" with this software. Instead of directly refering to a JS library offered thru standard CDN, he made the software "call home" to get the link.... Pretty strange way of doing it. And there's pretty much other strange trick like this !

It used to be a software that you'd need to pay or it would "break" after 30 days. Later on got "free" but with paying modules and now free. But if you look thru the code you may still see some part related to licensing.... There's copyright notice all over the place. Looks more like a publicity than anything else. There's too much superlative... Just look at the Push Request that user made and never got included...

**Comment by Undefined-Variables** [comment]
I agree with you @polynamaude... We have had many conversations about this and the issue I requested more information on in order to enable me to fix it never got answered! Wasted a lot of time on this software and had to move on!

**Reply by polynamaude** [reply]
@Undefined-Variables Are you still running your dating site for "specific needing customer" ?

**Reply by Nodrogchavez** [reply]
I've decided to just remove the entire program from my server, and look for something better.  From Softlab24 I got a very good social networking program.  It's about ready to release to the masses, and tech support is incredible as is the community support. You can see what I've put up at  https://communitycentral.ca

Cheers to all; time to move on.

**Reply by Undefined-Variables** [reply]
@polynamaude yes it’s still running and it’s doing well but I’m not adding anything else to it, I have moved on to something else.

**Reply by polynamaude** [reply]
@Undefined-Variables Sorry for you if it didn't go too far. Now you understand why I was saying "Yes" to a general dating site support but not a specific pH7CMS one ?

**Comment by polynamaude** [comment]
@Nodrogchavez I'm also in Canada (Quebec/Montreal).
I know other OpenSource dating software if you are looking out for something...
There's Una.IO too.... 
I've looked at the link you sent me. It look like a nice software but seems to me that there's a bit tricky... To sell for 20$ a module that allow you to "Ban user" is pretty strange. I always had the belief it was a standard and needed function !

**Reply by Nodrogchavez** [reply]
Hi polynamaude, from Vancouver, BC.

There is a Paid version for US$219 which includes much more than their free version.  The free one is substantial, but the Paid one is much more.  The Ban User is included in the Paid version.  I'm having a look at the UNA.IO to see if it interests me.

**Reply by Nodrogchavez** [reply]
Also, the site UNA.IO charges for things that other sites provide for free.  Example - https://una.io/searchKeyword.php?cat=bx_market_cats&keyword=15&section[]=bx_market

**Comment by polynamaude** [comment]
@Nodrogchavez Here new Canadian friend, make your eye bleed ( by the same author of this software )
https://lifyzer.com/

**Comment by Undefined-Variables** [comment]
@polynamaude I am happy with it now but to get it to it’s current state took me over a year of work so yes ph7cms was a good starting point if you are prepared to make it what you need it to be but the way it’s presented makes it look ready out of the box but the truth is it’s not ready. 

Yeah I understand what you mean and I’m still happy to be a part of a support forum for general dating website owners but I lost interest here due to the lack of support from the lead developer!

**Comment by polynamaude** [comment]
@Undefined-Variables We're similar ! I lost interest here and with Tiki CMS because it was soo chaotic.
Maybe we shall give OSSN a look ? What do you think ?

Did you repair your Minolta camera ?
![000076200003](https://user-images.githubusercontent.com/21186871/99729303-b4131380-2a88-11eb-9f86-95fd00920190.jpg)

**Comment by polynamaude** [comment]
![000076200002](https://user-images.githubusercontent.com/21186871/99729330-bffed580-2a88-11eb-9596-669478b5fd32.jpg)

**Comment by polynamaude** [comment]
![000076200006](https://user-images.githubusercontent.com/21186871/99729365-ce4cf180-2a88-11eb-82f6-1a1412d84fe8.jpg)

**Reply by Undefined-Variables** [reply]
@polynamaude Yeah camera is fixed! Brought a new laptop with a decent HD web cam too. Yeah I will have a look at it tomorrow, I will email you after I have looked...

**Comment by polynamaude** [comment]
@Undefined-Variables Great ! Those are picture I took on film of my GF.

**Reply by Undefined-Variables** [reply]
@polynamaude I love the pictures, like the way it’s b/w too 😃

**Comment by polynamaude** [comment]
@Nodrogchavez Wasn't sure what is included in the paid version of OSSN and why was there some options at $19/29/39...

A lifetime fee of 220$ is not much.

I'll check on the list of software I had for dating site and let you know what else I know about...

There's community builder (add on for Joomla) that can be great. Buddypress as addon for WordPress...

**Comment by polynamaude** [comment]
@Nodrogchavez Hi !
I registered on your site but never got the account confirmation email.

**Comment by norspeed22** [comment]
What ever happend to the OsDate srcipt, i still have the script, but have lost the database files so can`t re-install the script, does anyone have the database files for it ?

**Comment by hobb7t** [comment]
woohooo, there's a fire here, come on the Discord server guys, where can have live discussions about the dating webmastering thing https://discord.gg/p59BhR3AHe

**Comment by MAD-King** [comment]
Did you guys every found another dating software? Here, there is not support and the scripts are very buggy.
